backend/src/kitconcept/intranet/distributions/intranet/content/relations.json

```diff
@@ -319,61 +319,6 @@
     "from_uuid": "c8531174ccd54e39a284a347e9cece83",
     "to_uuid": "7ab9c48ede33415fa2a66a99549c8f70"
   },
-  {
```

Comment:
@tisto why is deleting so many references? did you remove any image?

Reply:
@sneridagh I didn't. No clue where this comes from.